docs/src/pages/core-concepts/component-hydration.md

```diff
@@ -95,6 +95,20 @@ To make your Astro component interactive, you will need to convert it to the fro
 
 Alternatively, you could add a `<script>` tag to your Astro component HTML template and send JavaScript to the browser that way. While this is fine for the simple stuff, we recommend a frontend framework for more complex interactive components.
 
+```astro
+---
+// Example: Using Astro with script tags
+// This is not recommended unless you are implementing very simple interactivity.
```

Comment:
I don't think we would want to include an example that's not recommended! In any case, I don't think this sample is bad at all. Indeed this might scale well, but I don't think that needs to be called out in this documentation.